service/administration/user.go

```diff
@@ -0,0 +1,466 @@
+	Tokens              []*Token      `json:"tokens,omitempty"`
+	PersonalAccessToken *string       `json:"personalAccessToken,omitempty"`
+	Id                  *int          `json:"id,omitempty"`
+
+	// forceSendFields is a list of field names (e.g. "Keys") to
+	// unconditionally include in API requests. By default, fields with
+	// empty values are omitted from API requests. However, any non-pointer,
+	// non-interface field appearing in ForceSendFields will be sent to the
+	// server regardless of whether the field is empty or not. This may be
+	// used to include empty fields in Patch requests.
+	forceSendFields []string
+
+	// nullFields is a list of field names (e.g. "Keys") to include in API
+	// requests with the JSON null value. By default, fields with empty
+	// values are omitted from API requests. However, any field with an
+	// empty value appearing in NullFields will be sent to the server as
+	// null. It is an error if a field in this list has a non-empty value.
+	// This may be used to include null fields in Patch requests.
+	nullFields []string
+}
+
+type UserPolicy struct {
+	PolicyId   *string `json:"policyId,omitempty"`
+	PolicyName *string `json:"policyName,omitempty"`
+	PolicyType *string `json:"policyType,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type ProgrammaticUser struct {
+	Name        *string       `json:"name,omitempty"`
+	Description *string       `json:"description,omitempty"`
+	Policies    []*ProgPolicy `json:"policies,omitempty"`
+	Accounts    []*Account    `json:"accounts,omitempty"`
+	Token       *string       `json:"token,omitempty"`
+	ProgUserId  *string       `json:"id,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type ProgPolicy struct {
+	PolicyId   *string  `json:"policyId,omitempty"`
+	AccountIds []string `json:"accountIds,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type Account struct {
+	Id   *string `json:"id,omitempty"`
+	Role *string `json:"role,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type Token struct {
+	Name            *string    `json:"name,omitempty"`
+	CreatedAt       *time.Time `json:"createdAt,omitempty"`
+	PolicyType      *string    `json:"policyType,omitempty"`
+	TokenId         *int       `json:"tokenId,omitempty"`
+	TokenLastDigits *string    `json:"tokenLastDigits,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type ListUsersInput struct{}
+
+type ListUsersOutput struct {
+	Users []*User `json:"users,omitempty"`
+}
+
+type CreateUserInput struct {
+}
+
+type CreateUserOutput struct {
+	User *User `json:"user,omitempty"`
+}
+
+type CreateProgrammaticUserInput struct {
+}
+
+type CreateProgrammaticUserOutput struct {
+	ProgrammaticUser *ProgrammaticUser `json:"user,omitempty"`
+}
+
+type ReadUserInput struct {
+	UserID *string `json:"userId,omitempty"`
+}
+
+type ReadUserOutput struct {
```

Comment:
get user details has groups object in the response. which is not covered in user struct